apps/hash-ai-worker-ts/src/activities/get-web-page-activity.ts

```diff
@@ -10,6 +10,49 @@ import { requestExternalInput } from "./shared/request-external-input";
 
 puppeteer.use(StealthPlugin());
 
+export const sanitizeHtmlForLlmConsumption = (params: {
+  htmlContent: string;
+  maximumNumberOfTokens?: number;
+}): string => {
+  const { htmlContent, maximumNumberOfTokens = 75_000 } = params;
+
+  const sanitizedHtml = sanitizeHtml(htmlContent, {
+    allowedTags: sanitizeHtml.defaults.allowedTags.filter(
+      /**
+       * @todo: consider whether there are other tags that aren't relevant
+       * for LLM consumption.
+       */
+      (tag) => !["script", "style", "link", "canvas", "svg"].includes(tag),
+    ),
+    allowedAttributes: {
+      "*": [
+        "href",
+        "src",
+        "onclick",
+        "title",
+        "alt",
+        "aria",
+        "label",
+        "aria-*",
+        "data-*",
+      ],
+    },
+    disallowedTagsMode: "discard",
+  });
+
+  const slicedSanitizedHtml = sanitizedHtml.slice(
+    0,
+    /**
+     * Assume that each token is 4 characters long.
+     *
+     * @todo: use `js-token` to more accurately determine the number of tokens.
```

Comment:
Is this the right library name?

Reply:
Sorry, you're right it should be `js-tiktoken`. Have addressed in https://github.com/hashintel/hash/pull/4505/commits/fb5423e94953d8ebb83ed9a38ae698e961242981